diff --git a/src/Feeds/Fetcher/Form/UploadFetcherForm.php b/src/Feeds/Fetcher/Form/UploadFetcherForm.php
index 3c821aa..08767b7 100644
--- a/src/Feeds/Fetcher/Form/UploadFetcherForm.php
+++ b/src/Feeds/Fetcher/Form/UploadFetcherForm.php
@@ -67,10 +67,10 @@ class UploadFetcherForm extends ExternalPluginFormBase implements ContainerInjec
    * {@inheritdoc}
    */
   public function validateConfigurationForm(array &$form, FormStateInterface $form_state) {
-    $values =& $form_state->getValues();
+    $values = $form_state->getValues();
 
     $values['allowed_extensions'] = preg_replace('/\s+/', ' ', trim($values['allowed_extensions']));
-
+    $form_state->setValue('allowed_extensions', $values['allowed_extensions']);
     // Ensure that the upload directory exists.
     if (!empty($form['directory']) && !file_prepare_directory($values['directory'], FILE_CREATE_DIRECTORY | FILE_MODIFY_PERMISSIONS)) {
       $form_state->setError($form['directory'], $this->t('The chosen directory does not exist and attempts to create it failed.'));
